tests/repository-fixture/index.js

```diff
@@ -0,0 +1,13 @@
+const sumTwoNumbers = (numA, numB) => {
```

Comment:
Y estos archivos qué onda????

No habrán sido un ejemplo?

Consejo muy muy muy muy recomendado:
SIEMPRE hacer un git diff antes de hacer un commit, para comprobar que no estamos comiteando cosas que no deberíamos estar comiteando.

Eso ninguna prueba ni ningún linter se va a dar cuenta y nos hace perder tiempo a todos.

Reply:
En realidad pense que este "demo" se debia subir para que las personas que se clonen el proyecto puedan correr los test, pero en este momento ya lo quite

Reply:
Ah bueno soy bien bruto, sí estaba perfecto como tenías jajaja. Sí, queremos que todo esté subido para poder correr los tests. No había entendido que ese era el repo de demo.
Así que vuelvalos a subir por fa, and sorry for that :pray: .